src/tox/config/loader/ini/factor.py

```diff
@@ -50,7 +50,7 @@ def expand_factors(value: str) -> Iterator[tuple[list[list[tuple[str, bool]]] |
     for line in value.split("\n"):
         factors: list[list[tuple[str, bool]]] | None = None
         marker_at, content = line.find(":"), line
-        if marker_at != -1:
+        if marker_at != -1 and (len(line) == marker_at + 1 or line[marker_at + 1] == " "):
```

Comment:
@gaborbernat  Should tabs be accepted too?

Reply:
Sure

Reply:
Submitted https://github.com/tox-dev/tox/pull/2823/